v2.0/admin-ui-access-and-navigate.md

```diff
@@ -56,7 +56,7 @@ Metric | Description
 --------|----
 Total Nodes | The total number of nodes in the cluster. <a href='admin-ui-access-and-navigate.html#decommissioned-nodes'>Decommissioned nodes</a> are not included in the Total Nodes count. <br><br>You can further drill down into the nodes details by clicking on [**View nodes list**](#nodes-list).
 Dead Nodes | The number of [dead nodes](admin-ui-access-and-navigate.html#dead-nodes) in the cluster.
-Capacity Used | The storage capacity used as a percentage of total storage capacity allocated across all nodes.<br><br>**Note:** If you are running multiple nodes on a single machine (not recommended), this value may be incorrect. This is because when multiple nodes are running on a single machine, the machine's hard disk is treated as separate stores for each node, while in reality, only one hard disk is used for all nodes. The Capacity Used is then displayed as the hard disk capacity used multiplied by the number of nodes on the machine.
+Capacity Used | The storage capacity used as a percentage of total storage capacity allocated across all nodes.<br /><br />**Note**: {% include available-capacity-metric.md %}
```

Comment:
It looks like using an include in the table does, in fact, break things: http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/ae5878f35d2552936f680b7306a56385dee2ebbf/stable/admin-ui-access-and-navigate.html#summary-panel

Can you play around with it and, if necessary, paste the text here directly?

Reply:
You could also just do the include after the table, as you've done in other places.

Reply:
Thanks for the catch!  I updated this page and the v1.1 version to do the include after the table in 7076917f.  Tables appear to be building correctly now "on my machine (tm)" :-)